src/test/resources/HW21.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+    <parameter name="BaseURL" value="https://bbb.testpro.io/"/>
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="HomeWork21"/>
+            <class name="BaseTest"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225427401-f77d9766-41be-4bce-89f8-2b5384e9c0e3.png)

HINT

For example

![Test status](https://user-images.githubusercontent.com/104761482/225427615-424a16bc-362d-474b-a9a9-5924579e803e.png)

![Test error_](https://user-images.githubusercontent.com/104761482/225427647-2de8aa1c-bbfa-4590-98fd-bc963a728638.png)